ieeg_epilepsy_ecog/sub-ecog01/ses-postimp/ieeg/sub-ecog01_ses-postimp_task-seizure_run-01_ieeg.vhdr

```diff
@@ -0,0 +1,119 @@
+Brain Vision Data Exchange Header File Version 1.0
+; Data created by Brainstorm
+
+[Common Infos]
+Codepage=UTF-8
+DataFile=070709aa_0027.eeg
+MarkerFile=070709aa_0027.vmrk
+DataFormat=BINARY
+; Data orientation: MULTIPLEXED=ch1,pt1, ch2,pt1 ...
+DataOrientation=MULTIPLEXED
+NumberOfChannels=97
+; Sampling interval in microseconds
+SamplingInterval=976.5625
+
+[Binary Infos]
+BinaryFormat=IEEE_FLOAT_32
+
+[Channel Infos]
+; Each entry: Ch<Channel number>=<Name>,<Reference channel name>,
+; <Resolution in "Unit">,<Unit>, Future extensions..
+; Fields are delimited by commas, some fields might be omitted (empty).
+; Commas in channel names are coded as "\1".
+Ch1=G_A1,,1,�V
```

Comment:
could it be that there is some encoding error going on here?

Reply:
Fixed